internal/cloud/realm/app.go

```diff
@@ -41,6 +41,11 @@ func (app App) Option() string {
 	return fmt.Sprintf("%s (%s)", app.ClientAppID, app.GroupID)
 }
 
+//IsZero checks for app GroupID and ID
```

Comment:
[nit] for the docstrings, make sure to place a space between `// ` and the start of the message

---

also, how many places is this actually used?  if its just a few times, you might be able to fix my bad code and just eliminate this method all together